CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # Version changelog
 
+## 0.23.0
+
+Note: this release includes custom JSON marshalling that captures whether or
+not a value is set and should be included in the marshalled output, if its
+value is equal to Go's zero value (i.e. `0` for an int field).
+
+* Add ForceSendFields and a custom marshaller ([#615](https://github.com/databricks/databricks-sdk-go/pull/615)).
+* Support text/plain response as streaming request body ([#638](https://github.com/databricks/databricks-sdk-go/pull/638)).
```

Comment:
nit:
```suggestion
* Added ForceSendFields and a custom marshaller ([#615](https://github.com/databricks/databricks-sdk-go/pull/615)).
* Supported text/plain response as streaming request body ([#638](https://github.com/databricks/databricks-sdk-go/pull/638)).
```

Reply:
Sticking to the PR titles; eval of past vs present tense should be discussed if you want it consistent.

Reply:
Yeah, I think usually we try to update PR Titles to be the past tense (before merge) but don't have a way update these automatically (if not past tense) while doing release.